js/interactive-guides/step-content.js

```diff
@@ -262,6 +262,35 @@ var stepContent = (function() {
     createEndOfGuideContent();
   };
 
+  var calculateWidgetHeight = function(percentageHeight) { 
+    var widgetHeight = "300px";
```

Comment:
Does this force the widget height for single-column view all to 300px?   What should we be doing here?